### PR TITLE
Search: webhooks migration script

### PR DESCRIPTION
**Pull request by krzysztofwolski**
## Scope of the PR

Add command line script to update webhooks

## Checklist

- [x] `.github/dependabot.yaml` is up-to date.
- [x] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: apps/search/scripts/migrations/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Webhook migration scripts
+
+Test migration with dry run, operation will not modify any data:
+`npx tsx scripts/migrations/run-webhooks-migration-dry-run.ts`
+
+To start the migration run command:
+`npx tsx scripts/migrations/run-webhooks-migration.ts`
```
Comment (lkostrowski) [inline apps/search/scripts/migrations/README.md:7]: note for upcoming hackathon solution - probably some npm scripts will do the job

File: apps/search/scripts/migrations/recreate-webhooks.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    .toPromise()
+    .then((r) => r.data?.app?.webhooks);
+
+  if (!webhooks?.length) {
+    console.error("The environment does not have any webhooks, skipping");
+    return;
+  }
+
+  // Use currently existing webhook data to determine a proper baseUrl and enabled state
+  const targetUrl = webhooks[0].targetUrl;
```
Comment (lkostrowski) [inline apps/search/scripts/migrations/recreate-webhooks.ts:34]: what if there is more than 1 webhook?

File: apps/search/scripts/migrations/recreate-webhooks.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  const enabled = webhooks[0].isActive;
+
+  const baseUrl = new URL(targetUrl).origin;
+
+  if (dryRun) {
+    console.log("Necessary data gathered, skipping recreation of webhooks due to dry run mode");
```
Comment (lkostrowski) [inline apps/search/scripts/migrations/recreate-webhooks.ts:40]: shouldnt' you print some report here?

File: apps/search/scripts/migrations/recreate-webhooks.ts (hunk 2 above)
Comment (krzysztofwolski) [reply apps/search/scripts/migrations/recreate-webhooks.ts:34]: I'm just using 2 things shared among all of the hooks - base URL and enabled status. The application always have the same values across all of the hooks.

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**